### PR TITLE
GEN-756 | Debugger: add hidden page debugger

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add hidden debug menu that you can access by pressing `?` on your keyboard

- Show ShopSession ID with option to copy to clipboard

## Justify why they are needed

- Allow us to present debug information for internal use

- We can expose things like shop session information or links

- I think we want to make this page specific so we can display information
  that is relevant to the page you are on

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2758** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2758" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2758?utm_source=stack-comment).

File: apps/store/src/pages/_app.tsx [hunk 1]
```diff
@@ -100,6 +101,7 @@ const MyApp = ({ Component, pageProps }: AppPropsWithLayout) => {
                 <BalancerProvider>
                   <AppErrorProvider>
                     <AppErrorDialog />
+                    <DebugDialog />
```
Comment (guilhermespopolin) [inline apps/store/src/pages/_app.tsx:104]: Should we render on in dev? 🤔

**Review by guilhermespopolin — APPROVED** [review]
Nice addition 👏🏻

File: apps/store/src/pages/_app.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/pages/_app.tsx:104]: I believe we can always render it -- but make sure that only people that should know about it can trigger it

**Comment by robinandeer** [comment]
@robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2758).

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2758).

**Pull request merged**